### PR TITLE
Front End

### PR DESCRIPTION
**Pull request by Satish196**

### CODE REVIEW

**Comment by Satish196** [comment]
**Front Page**
Screenshot from 2021-03-03 14-20-58

**Add New Person**
Screenshot from 2021-03-03 14-21-25

**After adding person**
Screenshot from 2021-03-03 14-21-31

**Update Person**
Screenshot from 2021-03-03 14-21-46

**After Update**
Screenshot from 2021-03-03 14-21-51

**View Single Person**
Screenshot from 2021-03-03 14-22-04

**After Delete**
Screenshot from 2021-03-03 14-22-13

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,108 @@
...
+      })
+  },[])
+
+  const deletePerson = (id) => {
+    axios.delete(`${apiUrl}/${id}`)
+      .then(res => {
+        const personsClone = [...persons]
+        const newPersonList = personsClone.filter(person => {
+          return person._id !== id
+        })
```
Comment (chirag-madlani) [inline src/App.js:32]: Filter will return new array. So no need to use spread operator here

**Review by chirag-madlani — CHANGES_REQUESTED** [review]
Overall looks good except one comment that i have added. Can you also try to accommodate  Add & Edit feature in a single component called AddEditPerson?

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,107 @@
...
+
+    axios.post(apiUrl, {
+      firstName,
+      lastName,
+      age,
+      email
+    }).then(res => {
+      const newPerson = res.data
+      const personsClone = [...persons]
+      setPersons([...personsClone, newPerson])
```
Comment (chirag-madlani) [inline src/App.js:51]: Sorry for missing this but this is redundant too, You can directly write
```
setPersons([...persons, newPerson])
```

**Review by chirag-madlani — APPROVED** [review]
Good Work

**Pull request merged**